src/libraries/Native/Windows/System.IO.Compression.Native/zlib-ng/zlib.h

```diff
@@ -36,7 +36,13 @@
 
 #include <stdint.h>
 #include <stdarg.h>
-#include "zconf.h"
+
+/* Added ZLIB_COMPAT check to choose between the two available zconf files */
+#if defined(ZLIB_COMPAT)
+#  include "zconf.h.in"
```

Comment:
I believe this should be zconf.h. The zconf.h.in file is input to CMake and configure.

Reply:
We typically don’t conditionally include zconf.h. Instead we conditionally include zlib.h/zlib-ng.h in our projects based on ZLIB_COMPAT.